src/index.js

```diff
@@ -35,20 +31,20 @@ if (utils.isIE11) {
 var error = debug('A-Frame:error');
 var warn = debug('A-Frame:warn');
 
-if (window.document.currentScript && window.document.currentScript.parentNode !==
+if (window.AFRAME_CONFIG.debug.warnIfNotHead && window.document.currentScript && window.document.currentScript.parentNode !==
     window.document.head && !window.debug) {
   warn('Put the A-Frame <script> tag in the <head> of the HTML *before* the scene to ' +
-       'ensure everything for A-Frame is properly registered before they are used from ' +
-       'HTML.');
+      'ensure everything for A-Frame is properly registered before they are used from ' +
+      'HTML.');
 }
 
 // Error if not using a server.
-if (window.location.protocol === 'file:') {
+if (window.AFRAME_CONFIG.debug.warnIfFileProtocol && window.location.protocol === 'file:') {
```

Comment:
Do we need configurable warnings? Can we just check if it's a cordova environment?